inverse/ft_inverse_sloreta.m

```diff
@@ -218,7 +218,7 @@
   end
 end
 
-L = cell2mat(sourcemodel.leadfield);
+L = cell2mat(sourcemodel.leadfield');
 G = L*L'; % Gram matrix
 invG = inv(G + lambda * eye(size(G))); % regularized G^-1
```

Comment:
Just realized that there is actually yet another problem with those lines: if I pass a precomputed filter, `ft_sourceanalysis` will delete `leadfield` from `sourcemodel`, hence these lines will fail. 
What is the best way to make sure they are only run if needed? 
An if-statement like `if ~hasfilter`?

Reply:
I attempted a fix for this, it works on my own data.